src/CMakeLists.txt

```diff
@@ -739,7 +748,7 @@ include_directories(SYSTEM ${Pugixml_INCLUDE_DIRS})
 list(APPEND LIBS ${Pugixml_LIBRARIES})
 
 if(NOT SOURCE_PACKAGE)
-  add_definitions(-Werror -Wfatal-errors )
+#  add_definitions(-Werror -Wfatal-errors )
```

Comment:
Is that some remaining debug code ?

Reply:
Yes, I did not mean to commit it. Will remove it.